assets/inference/environments/minimal-app-quickstart/environment.yaml

```diff
@@ -0,0 +1,13 @@
+image:
+  name: azureml/curated/minimal-app-quickstart
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+    - Dockerfile
+    - conda_dependencies.yaml
+    - entrypoint.sh
```

Comment:
These two don't technically need to be included here because they don't have any template tags like `{{latest-image-tag}}`. You could make a case that conda_dependencies.yaml could include them at some point, but not entrypoint.sh.